core/digital-twin-lib/src/DAppContract.ts

```diff
@@ -109,8 +109,12 @@ export class DAppContract {
   /**
    * Load the base information of the contract type.
    */
-  async loadBaseInfo(): Promise<void> {
-    await (this as any).ensureContract();
+  async loadBaseInfo(): Promise<boolean> {
+    try {
```

Comment:
Catching an error and throwing another Error is not necessary, because we are catching it within the loading component itself. So `return Promise.resolve(true)` and `return Promise.reject(new Error('Failed to load contract'));` can be just removed and ensureContract will trigger the same logic.

Reply:
Didn't we wanted to add some logic to check if the contract is deleted, to reduce amount of unnecessary loaded data.

```ts
const contract = runtime.contractLoader.loadContract('DataContract', address);
await runtime.executor.executeContractCall(contract, 'owner');
```

Reply:
>Didn't we wanted to add some logic to check if the contract is deleted, to reduce amount of unnecessary loaded data.

We realized, we need to keep it generic because the provided hash could be anything. We can't tell from a nulled data contract if it has been a twin in the past.